packages/react-internal/src/components/Modal/Modal.base.tsx

```diff
@@ -464,25 +413,25 @@ export class ModalBase extends React.Component<IModalProps, IDialogState> implem
         }
         case KeyCodes.up: {
           this.setState({
-            y: this._getClampedPositionY(this.state.y - delta),
+            y: this.state.y - delta,
```

Comment:
The clamping is a new change which hasn't been ported. We should definitely not remove that, and if it involved any new props to FocusTrapZone we should add those too.